RELEASE-NOTES.txt

```diff
@@ -1,5 +1,6 @@
 4.4
 -----
+ 
```

Comment:
nit: is this empty line intentional?

Reply:
It's not. I believe it's from the auto-merge conflict resolution. I also saw it before I opened the PR and considered removing it, but a commit to remove an empty line seemed unnecessary. We can remove it when we are editing the `RELEASE-NOTES.txt` for some other reason.

Thanks for the review!